tattoo.ts

```diff
@@ -79,16 +80,49 @@ const cli = cliArgs([
         "Set to clone all repos from remote instead of updating local copies."
   },
   {
+    name: "released",
+    type: Boolean,
+    defaultValue: false,
+    description:
+        "Set to update repos to the latest release when possible."
+  },
+  {
+    name: "branchconfig",
+    type: String,
+    defaultValue: "",
+    description:
+        "Set to use a config file to override branches/orgs for particular repos."
+  },
+  {
     name: "verbose",
     type: Boolean,
     defaultValue: false,
     description:
         "Set to print output from failed tests."
+  },
+  {
+    name: "wctflags",
```

Comment:
It needs a quoted argument - it doesn't pass flags through

Example: `--wctflags '-b chrome -p'`